acceptance_tests/features/view_collection_exercises_data.feature

```diff
@@ -13,11 +13,17 @@ Feature: View Collection Exercise
       | survey_id | survey_title              | survey_abbreviation | survey_legal_basis           |
       | 139       | Quarterly Business Survey | QBS                 | Statistics of Trade Act 1947 |
     And the internal user can view all collection exercises for QBS
-      | period | shown_to_respondent_as |
-      | 1803   | 9 March 2018           |
-      | 1806   | 15 June 2018           |
-      | 1809   | 14 September 2018      |
-      | 1812   | 14 December 2018       |
+      | period | shown_to_respondent_as | status  |
```

Comment:
I think we should have a separate scenario for the status

Reply:
I did have that originally but I discussed it with @rhiGriff and Harinder. They mentioned to add the scenario onto the existing scenario for viewing survey attributes, as its all part of the same page but to add an `And` for the viewing of states.

Reply:
I'd disagree with that justification, scenario testing isn't per page, it's per scenario or feature. If one feature fails only one scenario should fail this way it clear which features of the application are working or have regressed